--- conda/base/constants.py.orig	2016-12-08 01:27:04.837627916 +0000
+++ conda/base/constants.py	2016-12-08 01:32:09.634901005 +0000
@@ -76,6 +76,8 @@
                         "win-32",
                         "osx-64",
                         "linux-ppc64le",
+                        "linux-armv6l",
+                        "linux-armv7l",
                         "noarch",
                         )
 
