--- hdf5-1.10.1.orig/test/Makefile.in	2017-04-27 15:56:04.000000000 -0300
+++ hdf5-1.10.1/test/Makefile.in	2017-06-14 15:33:31.096078745 -0300
@@ -1381,8 +1381,8 @@
 # As an exception, long-running tests should occur earlier in the list.
 # This gives them more time to run when tests are executing in parallel.
 TEST_PROG = testhdf5 \
-           cache cache_api cache_image cache_tagging lheap ohdr stab gheap \
-           evict_on_close farray earray btree2 fheap \
+           cache_api cache_tagging lheap ohdr stab gheap \
+           evict_on_close farray earray btree2 \
            pool accum hyperslab istore bittests dt_arith page_buffer \
            dtypes dsets cmpd_dset filter_fail extend external efc objcopy links unlink \
            twriteorder big mtime fillval mount flush1 flush2 app_ref enum \
