From 71595bbd3b14d5936004035f0814726946b0c998 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Mon, 6 Jun 2016 17:08:18 +0100
Subject: [PATCH 3/3] Win64: Remove two SHORT annotations

---
 src/x86/win64.S | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/x86/win64.S b/src/x86/win64.S
index 687f97c..c47fbfc 100644
--- a/src/x86/win64.S
+++ b/src/x86/win64.S
@@ -170,7 +170,7 @@ ret_uint16$:
 	mov	rcx, QWORD PTR RVALUE[rbp]
 	movzx   rax, ax
 	mov	QWORD PTR [rcx], rax
-	jmp	SHORT ret_void$
+	jmp	ret_void$
 
 ret_sint16$:
  	cmp	DWORD PTR CIF_FLAGS[rbp], FFI_TYPE_SINT16
@@ -179,7 +179,7 @@ ret_sint16$:
 	mov	rcx, QWORD PTR RVALUE[rbp]
 	movsx   rax, ax
 	mov	QWORD PTR [rcx], rax
-	jmp	SHORT ret_void$
+	jmp	ret_void$
 
 ret_uint32$:
  	cmp	DWORD PTR CIF_FLAGS[rbp], FFI_TYPE_UINT32
-- 
2.8.2

