diff -Naur libffi-3.2.1.orig/include/Makefile.am libffi-3.2.1/include/Makefile.am
--- libffi-3.2.1.orig/include/Makefile.am	2014-11-12 10:00:59.000000000 -0200
+++ libffi-3.2.1/include/Makefile.am	2018-08-10 13:46:39.761552423 -0300
@@ -5,5 +5,5 @@
 DISTCLEANFILES=ffitarget.h
 EXTRA_DIST=ffi.h.in ffi_common.h
 
-includesdir = $(libdir)/@PACKAGE_NAME@-@PACKAGE_VERSION@/include
+includesdir = $(includedir)
 nodist_includes_HEADERS = ffi.h ffitarget.h
diff -Naur libffi-3.2.1.orig/libffi.pc.in libffi-3.2.1/libffi.pc.in
--- libffi-3.2.1.orig/libffi.pc.in	2014-11-12 10:00:59.000000000 -0200
+++ libffi-3.2.1/libffi.pc.in	2018-08-10 13:47:40.678693800 -0300
@@ -2,7 +2,7 @@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
 toolexeclibdir=@toolexeclibdir@
-includedir=${libdir}/@PACKAGE_NAME@-@PACKAGE_VERSION@/include
+includedir=@exec_prefix@/include
 
 Name: @PACKAGE_NAME@
 Description: Library supporting Foreign Function Interfaces
